core/src/main/java/nl/nn/adapterframework/http/authentication/OAuthAccessTokenManager.java

```diff
@@ -96,9 +102,8 @@ protected void abort() {
 
 		};
 		try (CloseableHttpResponse apacheHttpResponse = apacheHttpClient.execute(apacheHttpRequest)) {
-			String responseBody = StreamUtil.streamToString(apacheHttpResponse.getEntity().getContent(), null, null);
-			HTTPResponse httpResponse = convertFromApacheHttpResponse(apacheHttpResponse, responseBody);
-			parseResponse(httpResponse, responseBody);
+			HTTPResponse httpResponse = convertFromApacheHttpResponse(apacheHttpResponse);
+			parseResponse(httpResponse);
```

Comment:
In de huidige situatie werd responseBody als string meegegeven, om hem af te kunnen drukken in foutsituaties. Dat je niet alleen maar 'status 400' ziet, maar ook de foutmelding die in de body staat.
Ik denk dat je dat nu kwijt bent. Zou je dat kunnen testen?